python/pyspark/sql/types.py

```diff
@@ -189,7 +189,7 @@ def toInternal(self, dt):
         if dt is not None:
             seconds = (calendar.timegm(dt.utctimetuple()) if dt.tzinfo
```

Comment:
hmm, for the value of `epoch = datetime.datetime.fromtimestamp(0)`, `seconds` is 0. What is it different to use `int` or `long`?